sabnzbd/interface.py

```diff
@@ -1352,6 +1352,7 @@ def saveSwitches(self, **kwargs):
     "selftest_host",
     "rating_host",
     "ssdp_broadcast_interval",
+    "missing_threshold_mbytes",
```

Comment:
Could you move this after the `req_completion_rate`?
Both here and in `cfg.py`.